src/main/java/net/mcreator/ui/modgui/LivingEntityGUI.java

```diff
@@ -383,6 +385,7 @@ private synchronized void regenerateAITasks() {
 		});
 
 		spawnInDungeons.setOpaque(false);
+		spawnInRaids.setOpaque(false);

```

Comment:
I know you made it like `spawnInDungeons`, but since some times now, we try to group things a bit more, so maybe you should move it where you add the parameter to the panel (currently line #862).
(However, as this is only for the style, Klemen may tell you something else, so it would be best to wait for his answer on this comment.)

Reply:
Done